opendrift/models/physics_methods.py

```diff
@@ -527,15 +529,36 @@ def ftle(X, Y, delta, duration):
     J[:,:,1,0] = dy[0] / (2*delta)
     J[:,:,0,1] = dx[1] / (2*delta)
     J[:,:,1,1] = dy[1] / (2*delta)
-
+    
     for i in range(0,nx):
         for j in range(0,ny):
             # Green-Cauchy tensor
             D = np.dot(np.transpose(J[i,j]), J[i,j])
             # its largest eigenvalue
             lamda = np.linalg.eigvals(D)
             FTLE[i,j] = np.log(np.sqrt(max(lamda)))/np.abs(duration)
+    """
+    #from np.empty to np.zeros because new method doesn't work on boundaries. Empry left strange values.
+    FTLE = np.zeros([nx,ny])
+    for i in range(1,nx-1):
+        for j in range(1,ny-1):
+            #Centering grid on particle and calculating Jacobian of flow map
+            J = np.zeros([2,2])
+            J[0,0] = (X[i+1, j]-X[i-1, j])/(2*delta)
```

Comment:
This looks generally good. But is there a particular reason why you have moved calculation of Jacobian from outside of the loop and into the loop? I fear that this will give much longer computation time, as the overhead of each calculation is now repeated typically up to order of a million times (~if 1000 x 1000).
Did you notice such a slowdown?

Reply:
I see your point and yes, I noticed that it became somewhat slower. There should be a way to take it out of the loop, but I don't see how right now.